docs/pages/configuration/server_options.rst

```diff
@@ -1,44 +1,46 @@
+network.v1.collectmissingpayloadsinterval  60000                                                                                                                                                                                                                                                                                                            Interval (in milliseconds) that specifies how often the node should check for missing payloads and broadcast its peers for it (specify 0 to disable). This check might be heavy on larger DAGs so make sure not to run it too often.
+network.v2.gossipinterval                  5000                                                                                                                                                                                                                                                                                                             Interval (in milliseconds) that specifies how often the node should gossip its new hashes to other nodes.                                                                                                                           
+**VCR**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+vcr.jsonldcontexts.localmapping            [https://nuts.nl/credentials/v1=assets/contexts/nuts.ldjson,https://www.w3.org/2018/credentials/v1=assets/contexts/w3c-credentials-v1.ldjson,https://w3c-ccg.github.io/lds-jws2020/contexts/lds-jws2020-v1.json=assets/contexts/lds-jws2020-v1.ldjson,https://schema.org=assets/contexts/schema-org-v13.ldjson]  This setting allows mapping external urls to local files for e.g. preventing external dependencies. A source presented here has precedence over the remoteallowlist.                                                                
+vcr.jsonldcontexts.remoteallowlist         [https://schema.org,https://www.w3.org/2018/credentials/v1,https://w3c-ccg.github.io/lds-jws2020/contexts/lds-jws2020-v1.json]                                                                                                                                                                                   In strict mode, fetching external JSON-LD contexts is not allowed except for context-URLs listed here.                                                                                                                              
```

Comment:
https://schema.org doesn't directly resolve to a schema? Or do you need to provide an Accept-content-type?

Reply:
It's a bit weird, but the http header contains a link which points to the actual source:
`</docs/jsonldcontext.jsonld>; rel="alternate"; type="application/ld+json"` which is described here: https://w3c.github.io/json-ld-syntax/#alternate-document-location

Reply:
Lol, someone got creative. Didn't known you could do that.